src/components/Nav/Nav.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+function Nav(state) {
+  return (
+    <NavWrapper>
+      <NavLeft>
+        <button>
+          <i class="fas fa-bars"></i>
+          <span>프로젝트 둘러보기</span>
+        </button>
+        <button>프로젝트 올리기</button>
+      </NavLeft>
+      <NavCenter alt="logo" src="/images/logo.png" />
+      <NavRight>
+        <button>
+          <i class="fas fa-search"></i>
+        </button>
+        <button>로그인 / 회원가입</button>
+        <UserProfile />
+      </NavRight>
+    </NavWrapper>
+  );
+}
+
+const NavWrapper = styled.div`
+  ${({ theme }) => theme.flexSet('space-between')};
+  position: relative;
+  width: 100%;
+  height: 58px;
+  ${({ theme }) => theme.desktop`
+  width: 1032px;
+  height: 65px;
+  margin: 0px auto;
+  `};
```

Comment:
- 고정 높이 대신 margin, padding으로 높이를 주는게 좋을 것 같습니다.
- indent 맞춰주시면 더 보기 좋을 것 같습니다.
```suggestion
  ${({ theme }) => theme.desktop`
    width: 1032px;
    height: 65px;
    margin: 0px auto;
  `};
```

Reply:
네!